includes/fields/class-number.php

```diff
@@ -76,19 +83,25 @@ public static function init( $meta = [] ) {
 						'_order'    => 110,
 					],
 
-					'min_value'   => [
+					'min_value'          => [
 						'label'    => esc_html__( 'Minimum Value', 'hivepress' ),
 						'type'     => 'number',
 						'decimals' => 6,
 						'_order'   => 120,
 					],
 
-					'max_value'   => [
+					'max_value'          => [
 						'label'    => esc_html__( 'Maximum Value', 'hivepress' ),
 						'type'     => 'number',
 						'decimals' => 6,
 						'_order'   => 130,
 					],
+
+					'disable_formatting' => [
```

Comment:
Change it to a positive option by default, "formatted" that defaults to "true". It's ok, we'll fix the "checkboxes with the default value always get it" bug in a separate task.